opendevin/security/README.md

```diff
@@ -0,0 +1,57 @@
+# Security
+
+Given the impressive capabilities of OpenDevin and similar coding agents, ensuring robust security measures is essential to prevent unintended actions or security breaches. The SecurityAnalyzer framework provides a structured approach to monitor and analyze agent actions for potential security risks.
+
```

Comment:
Could you add a section here as a quick start to show people how to enable this feature? e.g., on the UI and on the backend?

Reply:
sure, added it in the latest commit:
````suggestion
To enable this feature: 
* From the web interface
    * Open Configuration (by clicking the gear icon in the bottom right)
    * Select a Security Analyzer from the dropdown
    * Save settings
    * (to disable) repeat the same steps, but click the X in the Security Analyzer dropdown
* From config.toml
```toml
[security]
# Enable confirmation mode
confirmation_mode = true
# The security analyzer to use
security_analyzer = "your-security-analyzer"
```
(to disable) remove the lines from config.toml
````

Reply:
Awesome!